client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -63,6 +65,23 @@ function ReaderSubscriptionListItem( {
 		return <ReaderSubscriptionListItemPlaceholder />;
 	}
 
+	function recordEvent( name ) {
```

Comment:
can you lift this out of the function so that it doesn't get recreated for each listItem? Will need to modify the signature to `recordEvent( name, { siteId: blog_ID, feedId: feed_ID, railcar } ) { `